modules/claims_api/app/workers/claims_api/special_issue_updater.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'sentry_logging'
+
+module ClaimsApi
+  class SpecialIssueUpdater
+    include Sidekiq::Worker
+    include SentryLogging
+
+    def perform(user, special_issues, auto_established_claim)
+      service = bgs_service(user).contention
+
+      contentions = service.find_contentions_by_ptcpnt_id(user.participant_id)
+
+      contention = contentions[:benefit_claims].select do |claim|
+                     claim[:clm_id] == auto_established_claim&.evss_id.to_s
+                   end [0]
+
+      options = contention_options(contention)
+
+      options[:contentions] =
+        [{
+          clm_id: contention[:clm_id],
+          special_issues: special_issues.map { |special_issue| { spis_tc: special_issue } }
+        }]
+
+      service.manage_contentions(options)
```

Comment:
theres a lot more data that is required for this method
https://github.com/department-of-veterans-affairs/bgs-ext/blob/ca867c91cf61afe775decd70d0e534c08a564480/spec/bgs/services/contention_spec.rb#L23
i was hoping most/all of it could be grabbed from the contentions call above? but line 22 doesnt contain any of that information. id assume this would raise an exception around required fields?

Reply:
I think that's what's getting added below in `contention_options`? Linting was complaining about method length and line 25 was the only thing that wasn't a straight 1:1 mapping, so I pulled all the other properties into that

Reply:
youre absolute correct, i somehow completely missed that method below :(